packages/utils/src/responsive.ts

```diff
@@ -28,33 +30,14 @@ export function mapResponsive(prop: any, mapper: (val: any) => any) {
 }
 
 export function objectToArrayNotation(obj: Dict) {
```

Comment:
I simplified this function a bit, please tell me if you prefer the previous version or if I've missed anything. I checked the tests to understand better what it was doing.

Reply:
This makes sense. Thank you